src/platform/plugins/shared/data_view_management/public/components/edit_index_pattern/edit_index_pattern.tsx

```diff
@@ -240,6 +240,7 @@ export const EditIndexPattern = withRouter(
               <>
                 <EuiSpacer />
                 <EuiCallOut
+                  announceOnMount
```

Comment:
```suggestion
                  announceOnMount={false}
```
Don't think this one changes after the page loads either.